packages/components/organisms/f-content-cards/src/components/cardTemplates/homePromotionCard/HomePromotionCard2.vue

```diff
@@ -118,14 +118,18 @@ export default {
         display: block;
         width: 100%;
         box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.1);
-        border-radius: $post-order-card-radius;
-        padding: spacing(x3) calc(35% + 8px) spacing(x3) spacing(x3);
+        border-radius: 4px;
```

Comment:
Just an FYI (as long as we can run it by design) – all border radiuses across the site are moving to 2px as part of PIE. So I'd advise changing any like this to use the `$border-radius` variable when we see them to save us some time further down the line, as long as design are happy for us to do so for this component.